apps/ledger-live-mobile/src/screens/BleDevicePairingFlow/index.tsx

```diff
@@ -71,16 +93,21 @@ export const BleDevicePairingFlow = ({
 }: BleDevicePairingFlowProps) => {
   const dispatchRedux = useDispatch();
 
+  const params = route?.params || {};
+
   const {
-    filterByDeviceModelId,
+    filterByDeviceModelId = undefined,
     areKnownDevicesDisplayed = true,
     onSuccessAddToKnownDevices = false,
-    onSuccessNavigateToConfig: {
-      navigateInput,
-      pathToDeviceParam,
-      navigationType = "navigate",
-    },
-  } = route.params;
+    onSuccessNavigateToConfig = defaultSuccessNavigateToConfig,
+  } = params;
+
+  const {
+    navigateInput,
+    pathToDeviceParam,
+    navigationType = "push",
```

Comment:
Wait - that's not what i was trying to say ahah

Here this default value for `navigationType` does not change anything, because you already set a default `navigationType` in `defaultSuccessNavigateToConfig`

And before, the `navigationType` was `navigate`, and now it is set to `push`, which changes the behavior of this component in other places where it is used.

Either put the default back to `navigate` inside `defaultSuccessNavigateToConfig` or make changes where this component was already used